asmcomp/arm64/emit.mlp

```diff
@@ -54,13 +54,13 @@ let emit_symbol s =
 (* Object types *)
 
 let emit_symbol_type emit_lbl_or_sym lbl_or_sym ty =
-  if not macosx then begin
+  if Config.asm_size_type_directives then begin
     `	.type	{emit_lbl_or_sym lbl_or_sym}, %{emit_string ty}\n`
   end
 
 
 let emit_symbol_size sym =
-  if not macosx then begin
+  if Config.asm_size_type_directives then begin
     `	.size	{emit_symbol sym}, .-{emit_symbol sym}\n`
   end
```

Comment:
Could/should these use the Emitaux helpers, to be more consistent with other backends?

Reply:
I considered it and didn’t do it (yet) as `arm64` overrides `emit_symbol`. I think the nicest way to handle this would be to modify the common `emit_symbol` to add the `_` on macOS.
For the `.type` directive, the `arm64` implementation is even more convoluted, as it emits `.type` for some labels (I wondered when that is useful). The idea I had (without implementing it yet) would be to use a wrapper for `.type` on labels and use the `Emitaux` one directly for other use cases. That would rely again on adding `_` to the general `emit_symbol`.

Reply:
Done in the last push.